csv2md/csv2md.py

```diff
@@ -37,13 +50,37 @@ def main():
                         help='delimiter character. Default is \',\'')
     parser.add_argument('-q', '--quotechar', metavar='QUOTECHAR', type=str, default='"',
                         help='quotation character. Default is \'"\'')
+    parser.add_argument('--center-fields', metavar='CENTER_FIELDS', type=str, default='',
+                        help='fields to center aligned')
+    parser.add_argument('--right-fields', metavar='RIGHT_FIELDS', type=str, default='',
+                        help='fields to right aligned')
```

Comment:
Please add short flags. Also I think it'll be more appropriate to use the word *column* instead of *field*. Maybe `--center-aligned-columns` and `--right-aligned-columns`.